src/resolver.rs

```diff
@@ -172,6 +173,7 @@ pub struct TypeAnnotator<'i> {
     pub(crate) annotation_map: AnnotationMapImpl,
     string_literals: StringLiterals,
     dependencies: IndexSet<Dependency>,
+    jumps_to_annotate: HashMap<String, HashMap<String, Vec<AstId>>>,
```

Comment:
I had to do some digging to understand what these entries in the HashMap resolve to (`<PouName, <LabelName, Vec<JumpStatementId>>>`), so maybe it is a good candidate for a very brief rustdoc comment?